tmt/convert.py

```diff
@@ -495,8 +501,14 @@ def target_content(target):
 
     # Nitrate (extract contact, environment and relevancy)
     if nitrate:
-        common_data, individual_data = read_nitrate(
-            beaker_task, data, disabled, general)
+        try:
+            common_data, individual_data = read_nitrate(
+                beaker_task, data, disabled, general)
```

Comment:
Have a slight concern: The `read_nitrate()` function is quite large one. Isn't capturing all exceptions a bit dangerous here? It could cover various other problems. Wouldn't it be better to cover the additional `nitrate.xmlrpc_driver.NitrateXmlrpcError` around here?

https://github.com/teemtee/tmt/blob/f056675a029e4802a6b7aa48c64c82c3e977fed0/tmt/convert.py#L543

If there is a problem with Nitrate connection it would pop up here.

Reply:
Well, expired kerberos ticket already raises ConvertError which is passed through.
Original exception is stored and printed so devel will know what is wrong.
However I don't think that user is interested - the outcome - nitrate not working is the same.

Reply:
Fixed as you proposed